aiohttp/web_app.py

```diff
@@ -48,8 +48,10 @@
 if TYPE_CHECKING:  # pragma: no cover
     _AppSignal = Signal[Callable[["Application"], Awaitable[None]]]
     _RespPrepareSignal = Signal[Callable[[Request, StreamResponse], Awaitable[None]]]
-    _Handler = Callable[[Request], Awaitable[StreamResponse]]
-    _Middleware = Callable[[Request, _Handler], Awaitable[StreamResponse]]
+
+    from .typedefs import Handler
```

Comment:
Import should probably go a couple of lines up before the declarations.

Reply:
Good call; fixed it.